WordPressUtils/build.gradle

```diff
@@ -19,15 +24,25 @@ repositories {
 
 dependencies {
     implementation 'org.apache.commons:commons-text:1.1'
-    implementation 'com.mcxiaoke.volley:library:1.0.18'
-    implementation 'com.android.support:design:28.0.0'
-    implementation 'com.android.support:recyclerview-v7:28.0.0'
+    implementation 'com.android.volley:volley:1.1.1'
+    implementation 'com.google.android.material:material:1.0.0'
+    implementation 'androidx.recyclerview:recyclerview:1.0.0'
     implementation 'org.greenrobot:eventbus:3.0.0'
 
+    implementation "androidx.core:core-ktx:+"
```

Comment:
I assume this was already reviewed and approved. However, I'd personally avoid using `+` in the build scripts. It slows down build time and more importantly we won't be able to easily find what version of the library was used in a certain release. Wdyt?

Reply:
I agree @malinajirka, but let's maybe do this in WPAndroid instead to avoid conflicts. I intend to open another PR just before creating a Peril rule to remind developers to update the utils library. We can include that fix in that PR.

I am also not planning to publish a new version to Bintray yet, so it shouldn't cause any issues elsewhere.

Reply:
Sounds like a great plan ;)!

Reply:
I was working on the `peril-settings` library today and had some extra time, so I opened a PR for this earlier than I expected to: https://github.com/Automattic/peril-settings/pull/50. I'll see if I can update this line as well as maybe the gradle dependencies for this project and WPAndroid next week.

Reply:
Nice! Great job, thanks!